web/src/modules/layout/components/GeneralHeader.js

```diff
@@ -13,22 +13,22 @@ type PropsType = {|
   t: TFunction
 |}
 
-class GeneralHeader extends React.PureComponent<PropsType> {
-  render () {
-    const { viewportSmall, t } = this.props
-    const getPath = new I18nRedirectRouteConfig().getRoutePath
+const GeneralHeader = ({ viewportSmall, t }: PropsType) => {
+  const getPath = new I18nRedirectRouteConfig().getRoutePath
 
-    const actionItems = !buildConfig().featureFlags.fixedCity
-      ? [<HeaderActionItemLink key='landing'
+  const actionItems = !buildConfig().featureFlags.fixedCity
+    ? [<HeaderActionItemLink key='landing'
                                href={getPath({})}
                                iconSrc={landingIcon}
                                text={t('changeLocation')} />]
-      : []
+    : []
 
-    return <Header viewportSmall={viewportSmall}
-                   logoHref={getPath({})}
-                   actionItems={actionItems} />
-  }
+  const onStickyTopChanged = useCallback(() => {}, [])
```

Comment:
What's that for?

Reply:
Required prop in the Header component which was not recognized previously because of untyped hocs. In order not to change the behaviour I use just a function doing nothing.